pom.xml

```
@@ -121,6 +121,26 @@
 			<version>2.24.3</version>
 		</dependency>
 
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt-api</artifactId>
+			<version>0.12.6</version>
```

same for other dependencies
```suggestion
			<version>${jjwt.version}</version>
```